diff --git a/docroot/.htaccess b/docroot/.htaccess
index efce1877..f52fe158 100755
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -1,3 +1,723 @@
+##
+#
+# Begin UW Medicine special .htaccess rules
+#
+#
+##
+<IfModule mod_rewrite.c>
+  RewriteEngine on
+
+  ## SSL RULES ##
+  # Redirect HTTP to HTTPS in Acquia Cloud
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} (ra|dev|test|prod)
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
+  # Set "protossl" to "s" if we were accessed via https://.  This is used later
+  # if you enable "www." stripping or enforcement, in order to ensure that
+  # you don't bounce between http and https.
+  RewriteRule ^ - [E=protossl]
+  RewriteCond %{HTTPS} on
+  RewriteRule ^ - [E=protossl:s]
+
+  # Make sure Authorization HTTP header is available to PHP
+  # even when running as CGI or FastCGI.
+  RewriteRule ^ - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+
+  ## UWMED CUSTOM REWRITES ##
+
+
+  # Allow simplesaml and UW NetId sign-ons to pass through
+  RewriteCond %{REQUEST_URI} !^/simplesaml
+
+  # Redirect CHEW Dev URL and CHEW UW URL to Live URL
+  RewriteCond %{HTTP_HOST} chew.cms.uwmedicine.org [OR]
+  RewriteCond %{HTTP_HOST} rightasrain.uw.edu
+  RewriteRule ^(.*)$ https://rightasrain.uwmedicine.org%{REQUEST_URI} [L,R=301]
+
+  # Redirect Huddle Dev URL to Live URL
+  RewriteCond %{HTTP_HOST} huddle.cms.uwmedicine.org
+  RewriteRule ^(.*)$ https://huddle.uwmedicine.org%{REQUEST_URI} [L,R=301]
+
+  # Redirect naked uwmedicine.org and uwmedicinetest.org to www version
+  RewriteCond %{HTTP_HOST} ^uwmedicine(test)?.org
+  RewriteCond %{HTTP_HOST} !^www\. [NC]
+  RewriteRule ^ http%{ENV:protossl}://www.%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
+  # SHAREPOINT RULES #
+  # Reverse proxy rules for launched pages. These paths should be served from SharePoint
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+  RewriteCond %{REQUEST_URI} ^/locations/.*/PublishingImages/.* [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-belltown/internal-medicine-residency-program/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-kent-des-moines/pediatric/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-kent-des-moines/residencia-pediatrica/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-northgate/family-medicine-residency-program/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/orcas-island-clinic/faqs/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/orcas-island-clinic/welcome/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/lopez-island-clinic/welcome/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/lopez-island-clinic/faqs/? [OR]
+  RewriteCond %{REQUEST_URI} ^/services/urgent-care/tips/?
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+  # DRUPAL RULES #
+  # Reverse proxy rules for launched pages. These paths should be served from Drupal
+  #
+  # Syntax for conditional matching: (/)?(?(1).*|$) - if we match / then allow .*, otherwise only allow $
+  # This prevents a match for something like /locations/mill-creeksdfg when using /?.*
+  #
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+  RewriteCond %{REQUEST_URI} !^/index.php$
+  RewriteCond %{REQUEST_URI} !^/(user|admin|themes|core|simplesaml|sites|libraries|profiles|modules|node|sitemaps|lp)(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/(views|editor|media_browser|toolbar|filter|contextual|history)/.+
+  RewriteCond %{REQUEST_URI} !^/?$
+  RewriteCond %{REQUEST_URI} !^/bios(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/patient-care/making-an-appointment$
+  RewriteCond %{REQUEST_URI} !^/patient-care/ecare$
+  RewriteCond %{REQUEST_URI} !^/services/primary-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/urgent-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/location-hours/.*
+  RewriteCond %{REQUEST_URI} !^/locations/adult-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/general-internal-medicine-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/international-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/lopez-island-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/madison-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-fremont-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-lake-forest-park-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-main-campus-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-north-seattle-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-outpatient-medical-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/orcas-island-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pioneer-square(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/primary-care([-a-zA-Z])(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/senior-care-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/third-avenue-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  # Start Neurosciences #
+  RewriteCond %{REQUEST_URI} !^/locations/alvord-brain-tumor-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cerebrovascular-lab-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/comprehensive-outpatient-rehabilitation-program-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eeg-neurophysiology-lab-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/gamma-knife-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/headache-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/multiple-sclerosis-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurodiagnostic-lab-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neuromuscular-respiratory-failure-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neuropsychological-assessment-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurosurgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/regional-epilepsy-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-rehabilitation-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/memory-wellness(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/neurosciences(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/neurological-surgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/neurology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/epilepsy(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/multiple-sclerosis(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/stroke(/)?(?(1).*|$)
+  # End Neurosciences #
+  # Start Urology and Men's Health #
+  RewriteCond %{REQUEST_URI} !^/locations/kidney-stone-center-nwhmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/mens-fertility(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/male-fertility-lab(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/mens-health-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-nw(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/mens-health(/)?(?(1).*|$)
+  # End Urology and Men's Health #
+  # Start Center for Women and Children
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/gynecology(/)?(?(1)(providers|locations|health-library|videos|approaches-to-gynecologic-care|family-planning|gynecologic-cancer|pediatric|urogynecology|robotic-assisted-gynecology|medical-discovery|grand-rounds-videos)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics(/)?(?(1)(providers|locations|childbirth-classes|pain-management|midwifery|birth-options|labor-and-delivery|approaches-to-obstetrics|you-and-your-baby|team|specialized-obstetrics|prenatal-diagnosis|lactation-services|grand-rounds-videos|nicu|diabetes-and-pregnancy|hypertension|teen-pregnancy|prematurity-prevention)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/reproductive(/)?(?(1)(providers|locations|services|team|health-library|referrals|single-women-lgbt|medical-discovery)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/urogynecology(/)?(?(1)(providers|locations|health-library|videos|medical-discovery)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/cancer-care/cancer-types/breast$
+  RewriteCond %{REQUEST_URI} !^/services/trauma-emergency-services/pediatric-trauma-and-burn-services$
+  RewriteCond %{REQUEST_URI} !^/locations/breast-health-scca(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/childbirth-center-northwest-hospital(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/childrens-care-neighborcare-columbia-city(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/childrens-care-neighborcare-meridian(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/family-planning-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/fertility-care-scca(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/gynonc-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/infant-development-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-and-infant-care-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-prenatal-diagnosis-clinic-arlington(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/obstetrics-arlington(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-prenatal-diagnosis-clinic-yakima(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-yvmh(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/birth-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-infant-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health-NWHMC(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nicu-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/midwives-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-trauma-burn-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pelvic-health-clinic-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pelvic-health-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/prenatal-diagnostic-springbrook(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/reproductive-health-fertility-clinic-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/reproductive-care-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/seattle-breast-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-of-seattle(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-center-scca(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-opmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-mill-creek(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sexual-assault-and-traumatic-stress-center-harborview(/)?(?(1).*|$)
+  # End Center for Women and Children #
+  # Start Eye Institute #
+  RewriteCond %{REQUEST_URI} !^/services/eye-institute(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eye-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eye-institute-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eyes-on-james(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/ophthalmology-harborview(/)?(?(1).*|$)
+  # End Eye Institue #
+  # Start ENT #
+  RewriteCond %{REQUEST_URI} !^/locations/audiology-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/audiology-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-otolaryngology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/ear-nose-and-throat-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/otolaryngology(/)?(?(1).*|$)
+  # End ENT #
+  # Start Mental Health Care #
+  RewriteCond %{REQUEST_URI} !^/locations/geropsychiatric-center-northwest-hospital(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/geropsychiatric-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-psychiatry-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-psychiatry-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/addictions-program-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatric-emergency-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatry-outpatient-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatry-consultation-service-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/mental-health-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/psychiatry(/)?(?(1).*|$)
+  # End Mental Health Care #
+  # Start Digestive Health Care #
+  RewriteCond %{REQUEST_URI} !^/locations/colorectal-services(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/digestive-health-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/endoscopy-suite-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/esophageal-and-gastric-diseases-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/gastroenterology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hepatitis-and-liver-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hepatology-clinic-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hepatology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-transplant-clinic-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-transplant-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-tumor-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/digestive-health-center-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/weight-loss-management-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/weight-regulation-diagnostic-and-evaluation-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/surgical-services-and-hernia-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/transplant/liver-transplant(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/digestive-health(/)?(?(1).*|$)
+  # End Digestive Health Care #
+  # Start Dermatology Care #
+  RewriteCond %{REQUEST_URI} !^/locations/dermatological-surgery-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/dermatology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/dermatology-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/dermatology(/)?(?(1).*|$)
+  # End Dermatology Care #
+  # Start Palliative Care #
+  RewriteCond %{REQUEST_URI} !^/services/palliative(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/palliative-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/palliative-care-outpatient-hmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/palliative-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/palliative-care-services-uwmc(/)?(?(1).*|$)
+  # End Palliative Care #
+ # Start Sleep Medicine #
+  RewriteCond %{REQUEST_URI} !^/services/sleep-institute(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/sleep-medicine(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/general-pulmonary-uwmc(/)?(?(1).*|$)
+  # End Sleep Medicine #
+  # Start Allergy & Immunology #
+  RewriteCond %{REQUEST_URI} !^/services/allergy-immunology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/allergy-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/allergy-asthma-and-immunology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/allergy-harborview(/)?(?(1).*|$)
+  # End Allergy & Immunology #
+  # Start Liver Transplant #
+  RewriteCond %{REQUEST_URI} !^/services/liver(/)?(?(1)(providers|locations|transplant|transplant/living-donor-liver-transplant|infectious-disease|patients-families|referrals|transplant-team)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-medical-specialties(/)?(?(1).*|$)
+  # End Liver Transplant #
+  # Start Laboratory Medicine & Pathology #
+  RewriteCond %{REQUEST_URI} !^/services/laboratory-medicine-and-pathology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-outpatient-medical-center(/)?(?(1).*|$)
+  # End Laboratory Medicine & Pathology #
+  # Start Vascular #
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-and-endovascular-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-at-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-surgery-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/wound-care-hyperbaric-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/vascular(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/vascular-care(/)?(?(1).*|$)
+  # End Vascular #
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+  ######################
+  ### UWMEDICINETEST ###
+  ######################
+  # Reverse proxy rules for stakeholder review pages. These should be served from Drupal.
+  RewriteCond %{HTTP_HOST} ^www.uwmedicinetest.org
+  RewriteCond %{REQUEST_URI} !^/index.php$
+  RewriteCond %{REQUEST_URI} !^/(user|admin|themes|core|simplesaml|sites|libraries|profiles|modules|node|sitemaps|lp)(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/(views|editor|media_browser|toolbar|filter|contextual|history)/.+
+  RewriteCond %{REQUEST_URI} !^/?$
+  RewriteCond %{REQUEST_URI} !^/patient-care/making-an-appointment$
+  RewriteCond %{REQUEST_URI} !^/patient-care/ecare$
+  RewriteCond %{REQUEST_URI} !^/bios(/)?(?(1).*|$)
+  # Start Primary Care #
+  RewriteCond %{REQUEST_URI} !^/services/primary-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/urgent-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/location-hours/.*
+  RewriteCond %{REQUEST_URI} !^/locations/adult-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/general-internal-medicine-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/international-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/lopez-island-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/madison-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-fremont-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-lake-forest-park-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-main-campus-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-north-seattle-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-outpatient-medical-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/orcas-island-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pioneer-square(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/primary-care([-a-zA-Z])(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/senior-care-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/third-avenue-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  # End Primary Care #
+  # Start Center for Women and Children
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/gynecology(/)?(?(1)(providers|locations|health-library|videos|approaches-to-gynecologic-care|family-planning|gynecologic-cancer|pediatric|urogynecology|robotic-assisted-gynecology|medical-discovery|grand-rounds-videos)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics(/)?(?(1)(providers|locations|childbirth-classes|pain-management|midwifery|birth-options|labor-and-delivery|approaches-to-obstetrics|you-and-your-baby|team|specialized-obstetrics|prenatal-diagnosis|lactation-services|grand-rounds-videos|nicu|diabetes-and-pregnancy|hypertension|teen-pregnancy|prematurity-prevention)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/reproductive(/)?(?(1)(providers|locations|services|team|health-library|referrals|single-women-lgbt|medical-discovery)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/cancer-care/cancer-types/breast$
+  RewriteCond %{REQUEST_URI} !^/services/trauma-emergency-services/pediatric-trauma-and-burn-services$
+  RewriteCond %{REQUEST_URI} !^/locations/breast-health-scca(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/childbirth-center-northwest-hospital(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/childrens-care-neighborcare-columbia-city(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/childrens-care-neighborcare-meridian(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/family-planning-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/fertility-care-scca(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/gynonc-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/infant-development-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-and-infant-care-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-prenatal-diagnosis-clinic-arlington(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/obstetrics-arlington(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-prenatal-diagnosis-clinic-yakima(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-yvmh(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/birth-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-infant-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health-NWHMC(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nicu-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/midwives-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-trauma-burn-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pelvic-health-clinic-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pelvic-health-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/prenatal-diagnostic-springbrook(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/reproductive-health-fertility-clinic-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/reproductive-care-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/seattle-breast-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-of-seattle(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-center-scca(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-opmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-mill-creek(/)?(?(1).*|$)
+  # End Center for Women and Children #
+  # Start Neurosciences #
+  RewriteCond %{REQUEST_URI} !^/locations/alvord-brain-tumor-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cerebrovascular-lab-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/comprehensive-outpatient-rehabilitation-program-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eeg-neurophysiology-lab-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/gamma-knife-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/headache-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/multiple-sclerosis-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurodiagnostic-lab-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neuromuscular-respiratory-failure-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neuropsychological-assessment-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurosurgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/regional-epilepsy-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-rehabilitation-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/memory-wellness(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/neurosciences(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/neurological-surgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/neurology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/epilepsy(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/multiple-sclerosis(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/stroke(/)?(?(1).*|$)
+  # End Neurosciences #
+  # Start Vascular #
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-and-endovascular-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-at-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-surgery-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/wound-care-hyperbaric-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/vascular(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/vascular-care(/)?(?(1).*|$)
+  # End Vascular #
+  # Start Urology and Men's Health #
+  RewriteCond %{REQUEST_URI} !^/locations/kidney-stone-center-nwhmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/mens-fertility(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/male-fertility-lab(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/mens-health-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/urology-nw(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/mens-health(/)?(?(1).*|$)
+  # End Urology and Men's Health #
+  # Start Mental Health Care #
+  RewriteCond %{REQUEST_URI} !^/locations/geropsychiatric-center-northwest-hospital(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/geropsychiatric-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-psychiatry-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-psychiatry-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/addictions-program-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatric-emergency-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatry-outpatient-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatry-consultation-service-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/mental-health-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/psychiatry(/)?(?(1).*|$)
+  # End Mental Health Care #
+  # Start Digestive Health Care #
+  RewriteCond %{REQUEST_URI} !^/locations/colorectal-services(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/digestive-health-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/endoscopy-suite-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/esophageal-and-gastric-diseases-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/gastroenterology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hepatitis-and-liver-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hepatology-clinic-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hepatology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-transplant-clinic-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-transplant-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-tumor-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/digestive-health-center-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/weight-loss-management-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/weight-regulation-diagnostic-and-evaluation-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/surgical-services-and-hernia-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/transplant/liver-transplant(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/digestive-health(/)?(?(1).*|$)
+  # End Digestive Health Care #
+  # Start Dermatology Care #
+  RewriteCond %{REQUEST_URI} !^/locations/dermatological-surgery-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/dermatology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/dermatology-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/dermatology(/)?(?(1).*|$)
+  # End Dermatology Care #
+  # Start ENT #
+  RewriteCond %{REQUEST_URI} !^/locations/audiology-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/audiology-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-otolaryngology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/ear-nose-and-throat-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/otolaryngology(/)?(?(1).*|$)
+  # End ENT #
+  # Start Heart Institute #
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-edmonds(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-imaging-echocardiography-ecg-nuclear-medicine-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/summit-cardiology-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/summit-cardiology-nwhmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cardiac-fitness-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/transplant-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/heart-institute(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/cardiology(/)?(?(1)(providers|locations|referrals-general-cardiology|cardiac-surgery|cardiac-electrophysiology|atrial-fibrillation|interventional-cardiology|adult-congenital-heart-disease|advanced-heart-failure|structural-heart-services|hypertrophic-cardiomyopathy|prosthetic-heart-valve-consult-clinic|cardio-oncology|cardiovascular-genetics)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/transplant/heart-transplant(/)?(?(1).*|$)
+  # End Heart Institute #
+  # Start Eye Institute #
+  RewriteCond %{REQUEST_URI} !^/services/eye-institute(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eye-center-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eye-institute-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/eyes-on-james(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/ophthalmology-harborview(/)?(?(1).*|$)
+  # End Eye Institue #
+  # Start Palliative Care #
+  RewriteCond %{REQUEST_URI} !^/services/palliative(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/palliative-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/palliative-care-outpatient-hmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/palliative-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/palliative-care-services-uwmc(/)?(?(1).*|$)
+  # End Palliative Care #
+  # Start Sleep Medicine #
+  RewriteCond %{REQUEST_URI} !^/services/sleep-institute(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/sleep-medicine(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/general-pulmonary-uwmc(/)?(?(1).*|$)
+  # End Sleep Medicine #
+  # Start Allergy & Immunology #
+  RewriteCond %{REQUEST_URI} !^/services/allergy-immunology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/allergy-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/allergy-asthma-and-immunology-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/allergy-harborview(/)?(?(1).*|$)
+  # End Allergy & Immunology #
+  # Start Laboratory Medicine & Pathology #
+  RewriteCond %{REQUEST_URI} !^/services/laboratory-medicine-and-pathology(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/clinical-lab-outpatient-medical-center(/)?(?(1).*|$)
+  # End Laboratory Medicine & Pathology #
+  # Start Liver Transplant #
+  RewriteCond %{REQUEST_URI} !^/services/liver(/)?(?(1)(providers|locations|transplant|transplant/living-donor-liver-transplant|infectious-disease|patients-families|referrals|transplant-team)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-medical-specialties(/)?(?(1).*|$)
+  # End Liver Transplant #
+  # Start Rehab Medicine #
+  RewriteCond %{REQUEST_URI} !^/services/rehabilitation-medicine(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/rehabilitation(/)?(?(1)(providers|locations|neuro-rehab-program|health-library|videos|inpatient|faq|patient-education)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/advanced-manual-therapy-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/advanced-manual-therapy-nwhmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/comprehensive-outpatient-rehabilitation-program-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/center-for-pain-relief-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/exercise-training-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/physical-therapy-services-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/physical-therapy-and-hand-therapy-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/prosthetics-and-orthotics(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/rehabilitation-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/rehabilitation-medicine-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-rehabilitation-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-rehabilitation-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/outpatient-rehabilitation-therapy-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/physical-sports-therapy-husky-stadium(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/physical-therapy-outpatient-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/brain-injury-rehabilitation-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/medical-rehabilitation-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hand-therapy-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/physical-occupational-therapy-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neuro-rehabilitation-northwest(/)?(?(1).*|$)
+  # End Rehab Medicine #
+  # Start Radiology #
+  RewriteCond %{REQUEST_URI} !^/services/radiology(/)?(?(1)(providers|locations|videos|appointments|radiation-safety|)|$)?(?(2).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/radiology-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/radiology-imaging-services-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/radiology-imaging-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/radiology-imaging-services-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/diagnostic-imaging-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-outpatient-imaging(/)?(?(1).*|$)
+  # End Radiolology #
+  # Start Cancer Care #
+  RewriteCond %{REQUEST_URI} !^/services/cancer-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/orthopedic-oncology-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/oncology-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/endocrine-tumor-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/cancer-center-radiation-oncology-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/plastic-and-reconstructive-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/thoracic-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/thoracic-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/breast-clinic-northwest(/)?(?(1).*|$)
+  # End Cancer Care #
+  # Start Orthopedic Surgery and Sports Medicine #
+  RewriteCond %{REQUEST_URI} !^/services/ssoh(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/orthopedic-surgery-sports-medicine(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/foot-ankle(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/hip-knee(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/hes(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/sports-medicine(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/advanced-manual-therapy-nwhmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/exercise-training-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/foot-and-ankle-center-hmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/foot-and-ankle-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hand-elbow-shoulder-center-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hand-elbow-shoulder-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hand-elbow-shoulder-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hip-and-knee-center-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hip-knee-center-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hip-knee-center-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hip-and-knee-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-hospital-foot-and-ankle-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/orthopaedic-trauma-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/orthopedic-oncology-center-uwmc-roosevelt(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/prosthetics-and-orthotics-eastlake(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/spine-center-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/spine-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-concussion-program-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-center-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/the-sports-medicine-clinic-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/the-sports-medicine-clinic-nwhmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/hand-elbow-nw(/)?(?(1).*|$)
+  # End Orthopedic Surgery and Sports Medicine #
+  # Start Spine Care #
+  RewriteCond %{REQUEST_URI} !^/services/spine(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/spine-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurosurgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/spine-center-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/spine-center-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-center-eastside(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/sports-medicine-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/the-sports-medicine-clinic-ballard(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/the-sports-medicine-clinic-nwhmc(/)?(?(1).*|$)
+  # End Spine Care #
+  # Start Burn Care #
+  RewriteCond %{REQUEST_URI} !^/services/burn-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/emergency-department-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/emergency-department-northwest(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/emergency-department-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/after-care-harborview-medical-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/burn-and-plastic-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/regional-burn-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/harborview/services/burn-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/trauma-emergency-services/getting-to-harborview(/)?(?(1).*|$)
+  # End Burn Care #
+  # Start Trauma Care #
+  RewriteCond %{REQUEST_URI} !^/services/trauma-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^harborview/services/trauma(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^harborview/services/trauma-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/orthopaedic-trauma-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/after-care-harborview-medical-center(/)?(?(1).*|$)
+  # End Trauma Care #
+  # Start Lung Care #
+  RewriteCond %{REQUEST_URI} !^/services/lung-care(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/lung(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/transplant/lung-transplant(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/chest-at-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/neuromuscular-respiratory-failure-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/general-pulmonary-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/bronchcoscopy-uwmc (/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/interstitial-lung-disease-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/lung-function-testing-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/lung-transplantation-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/nw-respiratory-associates(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/pulmonary-vascular-disease-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/thoracic-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/thoracic-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/transplant-services-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/adult-cystic-fibrosis-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/locations/tuberculosis-control-public-health-seattle-king-county-harborview(/)?(?(1).*|$)
+  # End Lung Care #
+  # Start Reconstructive & Plastic Surgery #
+  RewriteCond %{REQUEST_URI} !^/services/reconstructive-surgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^/services/surgery/reconstructive-and-plastic-surgery(/)?(?(1).*|$)
+  # End Reconstructive & Plastic Surgery #
+  # Start Surgery #
+  RewriteCond %{REQUEST_URI} !^/services/surgery(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/surgical-specialties-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/general-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/general-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/general-surgery-eastside-specialty-center(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/oral-maxillofacial-surgery-harborview(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/plastic-and-aesthetic-surgery-clinic(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/plastic-and-reconstructive-surgery-uwmc(/)?(?(1).*|$)
+  RewriteCond %{REQUEST_URI} !^locations/burn-and-plastic-surgery-harborview(/)?(?(1).*|$)
+  # End Surgery #
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+  # Redirects for launched pages. These pages are served from Drupal.
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine(test)?.org
+  RewriteRule ^locations/(.*)/directions(/)?(?(2).*|$) "https://www.uwmedicine.org/locations/$1#directions-tab" [NE,L,R=301]
+  RewriteRule ^locations/(.*)/care-providers(/)?(?(2).*|$) "https://www.uwmedicine.org/locations/$1#providers-tab" [NE,L,R=301]
+  RewriteRule ^services/urgent-care/(providers|locations)$ "https://www.uwmedicine.org/services/urgent-care" [L,R=301]
+  RewriteRule "^services/urgent-care/(.+)#911-calls$" "https://www.uwmedicine.org/services/urgent-care#block-emergencymedicine" [NE,L,R=301]
+  RewriteRule ^services/urgent-care/(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+
+</IfModule>
+##
+#
+#
+# END UWMedicine rules
+#
+#
+##
+
+
 #
 # Apache/PHP/Drupal settings:
 #
@@ -149,7 +869,11 @@ AddEncoding gzip svgz
   # Allow access to Statistics module's custom front controller.
   # Copy and adapt this rule to directly execute PHP files in contributed or
   # custom modules or to run another PHP application in the same directory.
-  RewriteCond %{REQUEST_URI} !/core/modules/statistics/statistics.php$
+  RewriteCond %{REQUEST_URI} !/core/modules/statistics/statistics.php$  
+
+  # UWM-CMS: Allow access to simplesaml paths
+  RewriteCond %{REQUEST_URI} !^/simplesaml
+
   # Deny access to any other PHP files that do not match the rules above.
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]
