diff --git a/docroot/.htaccess b/docroot/.htaccess
index efce1877..02e88d16 100755
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -1,3 +1,403 @@
+##
+#
+# Begin UW Medicine special .htaccess rules
+#
+#
+##
+<IfModule mod_rewrite.c>
+  RewriteEngine on
+
+  ## SSL RULES ##
+  # Redirect HTTP to HTTPS in Acquia Cloud
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} (ra|dev|test|prod)
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
+  # Set "protossl" to "s" if we were accessed via https://.  This is used later
+  # if you enable "www." stripping or enforcement, in order to ensure that
+  # you don't bounce between http and https.
+  RewriteRule ^ - [E=protossl]
+  RewriteCond %{HTTPS} on
+  RewriteRule ^ - [E=protossl:s]
+
+  # Make sure Authorization HTTP header is available to PHP
+  # even when running as CGI or FastCGI.
+  RewriteRule ^ - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+
+  ## UWMED CUSTOM REWRITES ##
+
+
+  # Allow simplesaml and UW NetId sign-ons to pass through
+  RewriteCond %{REQUEST_URI} !^/simplesaml
+
+  # Redirect CHEW Dev URL to Live URL
+  RewriteCond %{HTTP_HOST} chew.cms.uwmedicine.org
+  RewriteRule ^(.*)$ https://rightasrain.uwmedicine.org%{REQUEST_URI} [L,R=301]
+
+  # Redirect Huddle Dev URL to Live URL
+  RewriteCond %{HTTP_HOST} huddle.cms.uwmedicine.org
+  RewriteRule ^(.*)$ https://huddle.uwmedicine.org%{REQUEST_URI} [L,R=301]
+
+  # Redirect naked uwmedicine.org and uwmedicinetest.org to www version
+  RewriteCond %{HTTP_HOST} ^uwmedicine(test)?.org
+  RewriteCond %{HTTP_HOST} !^www\. [NC]
+  RewriteRule ^ http%{ENV:protossl}://www.%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
+  # SHAREPOINT RULES #
+  # Reverse proxy rules for launched pages. These paths should be served from SharePoint
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+  RewriteCond %{REQUEST_URI} ^/locations/.*/PublishingImages/.* [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-belltown/internal-medicine-residency-program/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-kent-des-moines/pediatric/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-kent-des-moines/residencia-pediatrica/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/primary-care-northgate/family-medicine-residency-program/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/orcas-island-clinic/faqs/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/orcas-island-clinic/welcome/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/lopez-island-clinic/welcome/? [OR]
+  RewriteCond %{REQUEST_URI} ^/locations/lopez-island-clinic/faqs/? [OR]
+  RewriteCond %{REQUEST_URI} ^/services/urgent-care/tips/?
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+  # DRUPAL RULES #
+  # Reverse proxy rules for launched pages. These paths should be served from Drupal
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+  RewriteCond %{REQUEST_URI} !^/index.php$
+  RewriteCond %{REQUEST_URI} !^/(user|admin|themes|core|simplesaml|sites|libraries|profiles|modules|node)/?.*
+  RewriteCond %{REQUEST_URI} !^/(views|editor|media_browser|toolbar|filter|contextual|history)/.+
+  RewriteCond %{REQUEST_URI} !^/?$
+  RewriteCond %{REQUEST_URI} !^/bios/?.*
+  RewriteCond %{REQUEST_URI} !^/patient-care/making-an-appointment$
+  RewriteCond %{REQUEST_URI} !^/patient-care/ecare$
+  RewriteCond %{REQUEST_URI} !^/services/primary-care/?.*
+  RewriteCond %{REQUEST_URI} !^/services/urgent-care/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/location-hours/.*
+  RewriteCond %{REQUEST_URI} !^/locations/adult-medicine-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/general-internal-medicine-center-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/international-medicine-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/lopez-island-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/madison-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-fremont-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-lake-forest-park-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-main-campus-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-north-seattle-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-outpatient-medical-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/orcas-island-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/pioneer-square/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/primary-care([-a-zA-Z])/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/senior-care-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/third-avenue-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt/?.*
+  # Start Neurosciences #
+  RewriteCond %{REQUEST_URI} !^/locations/alvord-brain-tumor-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cerebrovascular-lab-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/comprehensive-outpatient-rehabilitation-program-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/eeg-neurophysiology-lab-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/gamma-knife-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/headache-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/multiple-sclerosis-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurodiagnostic-lab-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neuromuscular-respiratory-failure-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neuropsychological-assessment-services-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurology/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurosurgery/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/regional-epilepsy-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-rehabilitation-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/memory-wellness/?.*
+  RewriteCond %{REQUEST_URI} !^/services/neurosciences/?.*
+  RewriteCond %{REQUEST_URI} !^/services/neurological-surgery/?.*
+  RewriteCond %{REQUEST_URI} !^/services/neurology/?.*
+  RewriteCond %{REQUEST_URI} !^/services/epilepsy/?.*
+  RewriteCond %{REQUEST_URI} !^/services/multiple-sclerosis/?.*
+  RewriteCond %{REQUEST_URI} !^/services/stroke/?.*
+  # End Neurosciences #
+  # Start Urology and Men's Health #
+  RewriteCond %{REQUEST_URI} !^/locations/kidney-stone-center-nwhmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/mens-fertility/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/male-fertility-lab/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/mens-health-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-nw/?.*
+  RewriteCond %{REQUEST_URI} !^/services/mens-health/?.*
+  # End Urology and Men's Health #
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+  # Reverse proxy rules for stakeholder review pages. These should be served from Drupal.
+  RewriteCond %{HTTP_HOST} ^www.uwmedicinetest.org
+  RewriteCond %{REQUEST_URI} !^/index.php$
+  RewriteCond %{REQUEST_URI} !^/(user|admin|themes|core|simplesaml|sites|libraries|profiles|modules|node)/?.*
+  RewriteCond %{REQUEST_URI} !^/(views|editor|media_browser|toolbar|filter|contextual|history)/.+
+  RewriteCond %{REQUEST_URI} !^/?$
+  RewriteCond %{REQUEST_URI} !^/patient-care/making-an-appointment$
+  RewriteCond %{REQUEST_URI} !^/patient-care/ecare$
+  RewriteCond %{REQUEST_URI} !^/bios/?.*
+  # Start Primary Care #
+  RewriteCond %{REQUEST_URI} !^/services/primary-care/?.*
+  RewriteCond %{REQUEST_URI} !^/services/urgent-care/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/location-hours/.*
+  RewriteCond %{REQUEST_URI} !^/locations/adult-medicine-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/general-internal-medicine-center-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/international-medicine-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/lopez-island-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/madison-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-fremont-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-lake-forest-park-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-main-campus-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-north-seattle-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-primary-care-outpatient-medical-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/orcas-island-clinic/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/pioneer-square/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/primary-care([-a-zA-Z])/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/senior-care-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/third-avenue-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt/?.*
+  # End Primary Care #
+  # Start Center for Women and Children
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children$
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children/maternity-care$
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children/childrens-care$
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children/breast-care$
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children/fertility-care$
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children/pelvic-care$
+  RewriteCond %{REQUEST_URI} !^/services/center-for-women-and-children/gynecology-care$
+  RewriteCond %{REQUEST_URI} !^/services/gynecology$
+  RewriteCond %{REQUEST_URI} !^/services/gynecology/approaches-to-gynecologic-care$
+  RewriteCond %{REQUEST_URI} !^/services/gynecology/family-planning$
+  RewriteCond %{REQUEST_URI} !^/services/gynecology/gynecologic-cancer$
+  RewriteCond %{REQUEST_URI} !^/services/gynecology/pediatric$
+  RewriteCond %{REQUEST_URI} !^/services/gynecology/urogynecology$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/childbirth-classes$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/pain-management$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/midwifery$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/birth-options$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/labor-and-delivery$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/approaches-to-obstetrics$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/team$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/prenatal-diagnosis$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/diabetes-and-pregnancy$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/hypertension$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/teen-pregnancy$
+  RewriteCond %{REQUEST_URI} !^/services/obstetrics/prematurity-prevention$
+  RewriteCond %{REQUEST_URI} !^/services/reproductive$
+  RewriteCond %{REQUEST_URI} !^/services/reproductive/services$
+  RewriteCond %{REQUEST_URI} !^/services/reproductive/team$
+  RewriteCond %{REQUEST_URI} !^/services/reproductive/referrals$
+  RewriteCond %{REQUEST_URI} !^/services/reproductive/single-women-lgbt$
+  RewriteCond %{REQUEST_URI} !^/services/cancer-care/cancer-types/breast$
+  RewriteCond %{REQUEST_URI} !^/services/trauma-emergency-services/pediatric-trauma-and-burn-services$
+  RewriteCond %{REQUEST_URI} !^/locations/breast-health-scca/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/childbirth-center-northwest-hospital/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/childrens-care-neighborcare-columbia-city/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/childrens-care-neighborcare-meridian/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/family-medicine-harborview/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/family-planning-uwmc-roosevelt/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/fertility-care-scca/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/gynonc-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/hall-health-center/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/infant-development-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-and-infant-care-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-prenatal-diagnosis-clinic-arlington/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/obstetrics-arlington/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-prenatal-diagnosis-clinic-yakima/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-fetal-medicine-yvmh/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/birth-center-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/maternal-infant-center/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health-ballard/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health-NWHMC/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/meridian-womens-health/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/nicu-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/midwives-clinic/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-care-center-uwmc-roosevelt/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/pediatrics-harborview/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/pediatric-trauma-burn-harborview/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/pelvic-health-clinic-eastside-specialty-center/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/pelvic-health-center-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/prenatal-diagnostic-springbrook/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/reproductive-health-fertility-clinic-roosevelt/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/reproductive-care-uwmc-roosevelt/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/seattle-breast-center/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-northwest/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-of-seattle/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-cancer-care-uwmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-center-scca/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-harborview/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-ballard/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-harborview/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-northwest/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-clinic-opmc/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-care-center-uwmc-roosevelt/?(directions|care-providers)?$
+  RewriteCond %{REQUEST_URI} !^/locations/womens-health-mill-creek/?(directions|care-providers)?$
+  # End Center for Women and Children #
+  # Start Neurosciences #
+  RewriteCond %{REQUEST_URI} !^/locations/alvord-brain-tumor-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cerebrovascular-lab-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/comprehensive-outpatient-rehabilitation-program-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/eeg-neurophysiology-lab-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/gamma-knife-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/headache-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/multiple-sclerosis-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurodiagnostic-lab-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurological-surgery-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neuromuscular-respiratory-failure-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neuropsychological-assessment-services-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/neurology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurology/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-neurosurgery/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/regional-epilepsy-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/sleep-medicine-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/stroke-rehabilitation-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/memory-wellness/?.*
+  RewriteCond %{REQUEST_URI} !^/services/neurosciences/?.*
+  RewriteCond %{REQUEST_URI} !^/services/neurological-surgery/?.*
+  RewriteCond %{REQUEST_URI} !^/services/neurology/?.*
+  RewriteCond %{REQUEST_URI} !^/services/epilepsy/?.*
+  RewriteCond %{REQUEST_URI} !^/services/multiple-sclerosis/?.*
+  RewriteCond %{REQUEST_URI} !^/services/stroke/?.*
+  # End Neurosciences #
+  # Start Vascular #
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-surgery-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-and-endovascular-surgery-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-at-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-laboratory-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/vascular-surgery-eastside-specialty-center/?.*
+  RewriteCond %{REQUEST_URI} !^/services/vascular/?.*
+  RewriteCond %{REQUEST_URI} !^/services/vascular-care/?.*
+  # End Vascular #
+  # Start Urology and Men's Health #
+  RewriteCond %{REQUEST_URI} !^/locations/kidney-stone-center-nwhmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/mens-fertility/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/male-fertility-lab/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/mens-health-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/urology-nw/?.*
+  RewriteCond %{REQUEST_URI} !^/services/mens-health/?.*
+  # End Urology and Men's Health #
+  # Start Mental Health Care #
+  RewriteCond %{REQUEST_URI} !^/locations/geropsychiatric-center-northwest-hospital/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/geropsychiatric-northwest/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-psychiatry-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/inpatient-psychiatry-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/addictions-program-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatric-emergency-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatry-outpatient-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/psychiatry-consultation-service-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/services/mental-health-care/?.*
+  RewriteCond %{REQUEST_URI} !^/services/psychiatry/?.*
+  # End Mental Health Care #
+  # Start Digestive Health Care #
+  RewriteCond %{REQUEST_URI} !^/locations/colorectal-services/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/digestive-health-center-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/endoscopy-suite-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/esophageal-and-gastric-diseases-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/gastroenterology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/hepatitis-and-liver-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/hepatology-clinic-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/hepatology-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-services-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/liver-transplant-clinic-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/liver-transplant-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/liver-tumor-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/digestive-health-center-eastside-specialty-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/liver-care-transplantation-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/nutrition-services-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/weight-loss-management-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/weight-regulation-diagnostic-and-evaluation-center-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/surgical-services-and-hernia-center/?.*
+  RewriteCond %{REQUEST_URI} !^/services/digestive-health/?.*
+  # End Digestive Health Care #
+  # Start Dermatology Care #
+  RewriteCond %{REQUEST_URI} !^/locations/dermatological-surgery-center-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/dermatology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/dermatology-uwmc-roosevelt/?.*
+  RewriteCond %{REQUEST_URI} !^/services/dermatology/?.*
+  # End Dermatology Care #
+  # Start ENT #
+  RewriteCond %{REQUEST_URI} !^/locations/audiology-services-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/audiology-services-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/northwest-otolaryngology/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-eastside/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-center-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/otolaryngology-head-and-neck-surgery-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/services/ear-nose-and-throat-care/?.*
+  RewriteCond %{REQUEST_URI} !^/services/otolaryngology/?.*
+  # End ENT #
+  # Start Heart Institute #
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-eastside-specialty-center/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-edmonds/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cardiology-imaging-echocardiography-ecg-nuclear-medicine-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/summit-cardiology-ballard/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/summit-cardiology-nwhmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/cardiac-fitness-northwest/?.*
+  RewriteCond %{REQUEST_URI} !^/services/heart-institute/?.*
+  RewriteCond %{REQUEST_URI} !^/services/cardiology/?.*
+  # End Heart Institute #
+  # Start Eye Institute #
+  RewriteCond %{REQUEST_URI} !^/services/eye-institute/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/eye-center-uwmc/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/eye-institute-harborview/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/eyes-on-james/?.*
+  RewriteCond %{REQUEST_URI} !^/locations/ophthalmology-harborview/?.*
+  # End Eye Institue #
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+  # Redirects for launched pages. These pages are served from Drupal.
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine(test)?.org
+  RewriteRule ^locations/(.*)/directions/?(.*)?$ "https://www.uwmedicine.org/locations/$1#directions-tab" [NE,L,R=301]
+  RewriteRule ^locations/(.*)/care-providers/?(.*)?$ "https://www.uwmedicine.org/locations/$1#providers-tab" [NE,L,R=301]
+  RewriteRule ^services/urgent-care/(providers|locations)$ "https://www.uwmedicine.org/services/urgent-care" [L,R=301]
+  RewriteRule "^services/urgent-care/(.+)#911-calls$" "https://www.uwmedicine.org/services/urgent-care#block-emergencymedicine" [NE,L,R=301]
+  RewriteRule ^services/urgent-care/(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+
+</IfModule>
+##
+#
+#
+# END UWMedicine rules
+#
+#
+##
+
+
 #
 # Apache/PHP/Drupal settings:
 #
@@ -150,6 +550,10 @@ AddEncoding gzip svgz
   # Copy and adapt this rule to directly execute PHP files in contributed or
   # custom modules or to run another PHP application in the same directory.
   RewriteCond %{REQUEST_URI} !/core/modules/statistics/statistics.php$
+
+  # UWM-CMS: Allow access to simplesaml paths
+  RewriteCond %{REQUEST_URI} !^/simplesaml
+
   # Deny access to any other PHP files that do not match the rules above.
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]
@@ -186,4 +590,4 @@ AddEncoding gzip svgz
   Header always set X-Content-Type-Options nosniff
   # Disable Proxy header, since it's an attack vector.
   RequestHeader unset Proxy
-</IfModule>
\ No newline at end of file
+</IfModule>
