diff -u vendor/simplesamlphp/simplesamlphp/metadata/saml20-idp-remote.php vendor/simplesamlphp/simplesamlphp/metadata-templates/saml20-idp-remote.php
--- vendor/simplesamlphp/simplesamlphp/metadata/saml20-idp-remote.php	2017-11-22 09:18:09.000000000 -0800
+++ vendor/simplesamlphp/simplesamlphp/metadata-templates/saml20-idp-remote.php	2017-11-22 09:18:30.000000000 -0800
@@ -22,3 +22,18 @@
 	'certFingerprint'      => 'c9ed4dfb07caf13fc21e0fec1572047eb8a7a4cb'
 );
 
+
+ 
+/***
+ * Include UWMCMS configurations.
+ *
+ ***/
+$config_file = sprintf('/mnt/gfs/%s.%s/nobackup/cmsdev.uwmedicine.org/simplesamlphp/metadata/saml20-idp-remote.php',
+    $_ENV['AH_SITE_GROUP'],
+    $_ENV['AH_SITE_ENVIRONMENT']);
+if (!file_exists($config_file) || !require_once($config_file)) {
+
+    exit('Missing metadata file '. $config_file);
+
+}
+
