diff -u vendor/simplesamlphp/simplesamlphp/config/authsources.php vendor/simplesamlphp/simplesamlphp/config-templates/authsources.php
--- vendor/simplesamlphp/simplesamlphp/config/authsources.php	2017-11-22 09:12:58.000000000 -0800
+++ vendor/simplesamlphp/simplesamlphp/config-templates/authsources.php	2017-11-22 09:16:06.000000000 -0800
@@ -390,3 +390,17 @@
     */
 
 );
+
+
+/***
+ * Include UWMCMS configurations.
+ *
+ ***/
+$config_file = sprintf('/mnt/gfs/%s.%s/nobackup/cmsdev.uwmedicine.org/simplesamlphp/config/authsources.php',
+    $_ENV['AH_SITE_GROUP'],
+    $_ENV['AH_SITE_ENVIRONMENT']);
+if (!file_exists($config_file) || !require_once($config_file)) {
+
+    exit('Missing config file '. $config_file);
+
+}
\ No newline at end of file
diff -u vendor/simplesamlphp/simplesamlphp/config/config.php vendor/simplesamlphp/simplesamlphp/config-templates/config.php
--- vendor/simplesamlphp/simplesamlphp/config/config.php	2017-11-22 09:12:58.000000000 -0800
+++ vendor/simplesamlphp/simplesamlphp/config-templates/config.php	2017-11-22 09:20:16.000000000 -0800
@@ -849,3 +849,17 @@
     'trusted.url.domains' => array(),
 
 );
+
+ 
+/***
+ * Include UWMCMS configurations.
+ *
+ ***/
+$config_file = sprintf('/mnt/gfs/%s.%s/nobackup/cmsdev.uwmedicine.org/simplesamlphp/config/acquia_config.php',
+    $_ENV['AH_SITE_GROUP'],
+    $_ENV['AH_SITE_ENVIRONMENT']);
+if (!file_exists($config_file) || !require_once($config_file)) {
+
+    throw new Exception('Missing config file '. $config_file);
+
+}
